packages/flame-css/README.md

```diff
@@ -0,0 +1,36 @@
+# Flame css
+
+`flame-css` is an alternative offering to `flame`.
+
+It offers a sass codebase and an easy to consume css file.
+
+@TODO:
+
+- [ ] Hook flame-tokens/flame-foundations as primary variables
+- [ ] Integrate tailwind to output utility classes
+- [x] Storybook setup
+- [x] Write all stories
+- [ ] Snapshot testing
+- [ ] Publishing
+
+## Caveat
+
+Although we are bundling the sass source code, for the most part, you should only be using the generated css file.
```

Comment:
```suggestion
Although we are bundling the Sass source code,  you should only be using the generated CSS file in the majority of cases.
```

Ah, here's what I mentioned in the README intro. I would move this at the top so that it's clear from the start.